vendor/github.com/pmorie/go-open-service-broker-client/v2/client.go

```diff
@@ -1,3 +1,4 @@
+// TODO (nilebox): Revert and update the dependency after PR merged: https://github.com/pmorie/go-open-service-broker-client/pull/50
```

Comment:
This is a temporary copy-pasta until https://github.com/pmorie/go-open-service-broker-client/pull/50 gets merged.